app/models/asset_field_type.rb

```diff
@@ -52,6 +53,29 @@ def mapping
 
   private
 
+  def is_image(mime_type)
```

Comment:
Also - when writing a boolean method, please add `?` after the name for self-documentation on what it returns